tests/integ/compiler/test_query_generator.py

```diff
@@ -323,16 +341,10 @@ def test_dataframe_creation_with_multiple_queries(session):
 
     verify_multiple_create_queries(queries, post_actions, 3)
 
-    query_generator = create_query_generator(df._plan)
-    # reset the whole plan
-    reset_plan_tree(df._plan)
-    # regenerate the queries
-    plan_queries = query_generator.generate_queries([df._plan.source_plan])
-    queries = [query.sql.lstrip() for query in plan_queries[PlanQueryType.QUERIES]]
-    post_actions = [
-        query.sql.lstrip() for query in plan_queries[PlanQueryType.POST_ACTIONS]
-    ]
-    verify_multiple_create_queries(queries, post_actions, 3)
```

Comment:
since i changed to use re_resolve_and_compare_plan_queries which will re-resolve every intermediate state and do exact compare, this verify is redundant since we already do such verification for the original generated query.